### DISCUSSION TITLE
AWS CloudFormation Fails - The following resource(s) failed to create: [KongScalingGroup].

### DISCUSSION THREAD

**Discussion by aaronfagan** [Kong/kong#7229] [Help]
### Summary

When attempting to create Kong following the directions here: https://docs.konghq.com/install/aws-cloudformation/ the CloudFormation fails. We are attempting to create in `us-west-2` with latest version of Kong. The following error appear on the `KongScalingGroup` step:

```
KongScalingGroup | CREATE_FAILED | Received 1 FAILURE signal(s) out of 2. Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
```

### Steps To Reproduce

1. Click to link to take you to your AWS account in `us-west-2` on the CloudFormation configure screen. Ensure you have a keypair available for use (we are using a 4096 bit pair)
2. Click next until creation is initiated, wait for evens
3. Failure event should appear.

### Additional Details & Logs

- No other logs available apart from what appears under CloudFormation stack > Events.

**Comment by sfw185** [comment]
@aaronfagan Have you since resolved this? Encountering the same issue at the moment, and tried with a few of the stack definitions (same result).
